src/components/views/rooms/LegacyRoomHeader.tsx

```diff
@@ -669,6 +672,19 @@ export default class RoomHeader extends React.Component<IProps, IState> {
 
         return (
             <>
+                {this.props.additionalButtons?.map(({ icon, id, label, onClick }) => (
+                    <Tooltip label={label()} key={id}>
```

Comment:
How do you feel about calling `label()` twice? Is it possible a module developer might be surprised that their callback was called multiple times?

Reply:
I think that it is an amazing hint of yours. Thanks for the catch! To be safe `label()` should be called once.

Reply:
Awesome, thank you!